fcp/src/tests/conftest.py

```
@@ -0,0 +1,59 @@
+@pytest.fixture
+def sample_metavariables() -> str:
+    return SAMPLE_METAVARIABLES
+
+
+@pytest.fixture
+def sample_stderr() -> str:
+    return SAMPLE_STDERR
+
+
+@pytest.fixture
+def sample_specification(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)